Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 
 name = "rpassword"
-version = "3.0.2"
+version = "3.0.3"
```

Comment:
Given that the `winapi` version is changing, I think there is a breaking change, because we can't compile `rpassword` with a project that uses `winapi` version `0.2` anymore, right ? If so, we'd have to bump up `rpassword` to version `4.0.0` to comply with semver.

Reply:
Yeah I think that's correct, since it might not be backwards compatible.